west.yml

```diff
@@ -61,7 +61,7 @@ manifest:
     # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/modules.html
     - name: zephyr
       repo-path: sdk-zephyr
-      revision: a8dec43b92d373e00bc33ef957c0c7bc5f6989d1
+      revision: 815560169829ee2cb7b4aaf89c6e677554402cd6
```

Comment:
@tmon-nordic can  you please update to https://github.com/nrfconnect/sdk-zephyr/commit/ea1fd932d014b49f27482463ce291b5697df9b38, so we can drop manifest updates in https://github.com/nrfconnect/sdk-nrf/pull/14061 and https://github.com/nrfconnect/sdk-nrf/pull/13935